Tasks/XcodeV5/xcodeutils.ts

```diff
@@ -145,6 +145,13 @@ function getPbxProjectPath(workspace: string) {
     if (workspace && workspace.trim().toLowerCase().endsWith('.xcworkspace')) {
         let pbxProjectPath: string = workspace.trim().toLowerCase().replace('.xcworkspace', '.pbxproj');
 
+        if (pathExistsAsFile(pbxProjectPath)) {
+            return pbxProjectPath;
+        }
+
+        tl.debug("Corresponding pbxProject file doesn't exist: " + pbxProjectPath + ", checking alternative location");
+        
+        pbxProjectPath = workspace.trim().toLowerCase().replace('.xcworkspace', '.xcodeproj/project.pbxproj');
```

Comment:
Would it be better to use another variable instead of overwriting the existing one?